src/components/Squiggly.module.css

```diff
@@ -33,6 +33,6 @@
 	transform: scale(1 1.17);
 }
 
-.squiggly:is(.alwaysActive, .passive, :focus, :hover):not(.inactive)::after {
+.squiggly:is(.alwaysActive, .passive, :hover):not(.inactive)::after {
```

Comment:
[Bug] I tried this out using only a keyboard and, as expected, the native focus indicator still came into play. So this passes accessibility needs! Yay!

But: I do think it's nice for non-mouse folks to still see the squiggly on focus. With this change as-is they don't get it:

![Screenshot of the 'Book' and 'Blog' header links, with native keyboard focus around 'Book'. No squiggly.](https://github.com/JoshuaKGoldberg/dot-com/assets/3335181/03de97c7-8dd8-4e0b-9548-fde66710dde6)

Proposal: how about switching this to [`:focus-visible`](https://developer.mozilla.org/en-US/docs/Web/CSS/:focus-visible)? That way keyboard folks get it, but mouse-only folks don't.

```suggestion
.squiggly:is(.alwaysActive, .passive, :focus-visible, :hover):not(.inactive)::after {
```

Reply:
Yes, I think that's the true solution that maintains the asthetic you want but fixes the Chromium-specific ui bug. Awesome revision. Now the behavior is uniform across Safari and Chromium (and FF). The keyboard users still get the squiggly, but navigating to the external site then coming back doesn't show the squiggly artifact on that Book link on Chromium.

Reply:
Yeah per https://caniuse.com/?search=focus%20visible `:focus-visible` support is pretty recent. Good timing on this 😄